### DISCUSSION TITLE
How to use hindsight experience replay in skrl?

### DISCUSSION THREAD

**Discussion by Aqua12138** [Toni-SM/skrl#29] [Q&A]
I want to know how to use hightsight experience replay in skrl

**Comment by Toni-SM** [comment]
Hi @Aqua12138 

HER implementation is already on the roadmap for future skrl implementations (https://github.com/users/Toni-SM/projects/2/views/8). With the current skrl version is not possible to apply HER, at least not directly, Although I need to find out more I think it would not be difficult to extend, given the modularity of the library, the implementation of such a feature. I will try to include it in the next planned release (> 0.8.0, already in progress)

**Reply by psh9002** [reply]
Hi @Toni-SM 

Is HER implementation included latest version?

**Reply by Toni-SM** [reply]
Hi @psh9002 

It is a pending task that I want to address as soon as possible.

**Reply by psh9002** [reply]
Thanks, @Toni-SM 

I look forward to being update!👍
